source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -248,10 +254,19 @@ def get_places(self, obj):
             'public_transport:stop_area': GeocodeStopAreaSerializer
         }
         res = []
-        for feature in obj.get('features', {}):
+        for feature in obj.get('features', []):
             type_ = feature.get('properties', {}).get('geocoding', {}).get('type')
             if not type_ or type_ not in map_serializer:
                 logging.getLogger(__name__).error('Place not serialized (unknown type): {}'.format(feature))
                 continue
             res.append(map_serializer[type_](feature).data)
         return res
+
+    def get_feed_publishers(self, obj):
+        fp = []
+        for feature in obj.get('features', []):
+            if feature.get('properties', {}).get('geocoding', {}).get('feed_publishers'):
```

Comment:
Maybe you can extract the long getter in a separate variable, to avoid duplication and make it more readable like: 
```python
feed_pubs = feature.get('properties', {}).get('geocoding', {}).get('feed_publishers')
if feed_pubs :
 [fp.append(x) for x in feed_pubs if x not in fp]

Reply:
Done